mypy.ini

```diff
@@ -1,4 +1,5 @@
 [mypy]
+python_version = 3.7
```

Comment:
This setting enables us to test against Python 3.7 typing semantics directly, simplifying local running and CI.

Reply:
Does this mean that we now _never_ test against 3.11?

Reply:
It's in the example config from `mypy`, I think of it like the version we use in `black`.    Unless there is a backwards-incompatible change to typing, this should be fine.

Reply:
Okay sounds good to me.